main.js

```diff
@@ -8,9 +8,51 @@
         attribution: '&amp;copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
     }).addTo(map);
 
+    //Array of markers and a parallel array that holds their date of occurance
```

Comment:
This works, but I think we can do better. Two arrays seems unnecessary. What if we just had a marker array that was an array of objects containing a `.pin` property that could be the `L.marker` object, in addition to the rest of the `record` that comes back from the WPRDC `fetch` call.

It's less memory, and we don't have to worry about making sure that the two array indexes match up. It's less moving parts. I'll go into what I'm talking about below